src/main/java/wheresmymoney/ExpenseList.java

```diff
@@ -32,26 +32,35 @@ public boolean isEmpty() {
         return expenses.isEmpty();
     }
 
-    public Expense getExpenseAtIndex(int i) {
-        return expenses.get(i);
+    public Expense getExpenseAtIndex(int i) throws WheresMyMoneyException {
```

Comment:
instead of using "i" in your params, use the full word "index"

Reply:
changed